app/views/posts/_post.html.haml

```diff
@@ -0,0 +1,33 @@
+.col-md-4{:id => "post_#{post.id}"}
+  .panel.panel-primary
+    .panel-heading
+      .panel-title
+        =post.title
+    .panel-body
+      =post.description
+    .panel-footer
+      %button.btn.btn-warning{"data-target" => "#updatepost_#{post.id}", "data-toggle" => "modal", :type => "button"}
+        Edytuj
+      = link_to 'Destroy', [@board,post], method: :delete, class: "btn btn-danger", remote: true
+
+=form_for [@board,post], :method => :put, remote: true do |f|
+  .modal.fade{"aria-labelledby" => "myModalLabel", :id => "updatepost_#{post.id}", :role => "dialog", :tabindex => "-1"}
```

Comment:
Bardzo spoko, tylko w tym momencie dla każdego wyświetlonego posta renderujesz modal, w dodatku z tym samym id-kiem. Przeróbmy to tak, że nie renderujemy formy, i dopiero jak ktoś kliknie 'edytuj', przez akcję ajaxową renderujemy ten form, pokazujemy modal, i po udanym updacie wypieprzamy go z DOM-a, żeby nie bruździł.